gateways/firstdata/types.go

```diff
@@ -0,0 +1,149 @@
+package firstdata
+
+// RequestType represents the valid requestType values that can be sent in a firstdata request
+type RequestType string
+
+const (
+	RequestTypeAuth    RequestType = "PaymentCardPreAuthTransaction"
```

Comment:
just checking - sale transaction means no capture, right?

Reply:
Yeah, PreAuth first, then PostAuth to capture